pkg/oci/utils.go

```diff
@@ -286,6 +287,35 @@ func (spec *CompatOCISpec) PodID() (string, error) {
 	return "", fmt.Errorf("Could not find pod ID")
 }
 
+func ociResourceUpdateRuntimeConfig(ocispec CompatOCISpec, config *RuntimeConfig) error {
+	if ocispec.Linux == nil {
```

Comment:
Make sure you gather all the statements together as they end up with the same result:
```go
if ocispec.Linux == nil ||
   ocispec.Linux.Resources == nil ||
   ocispec.Linux.Resources.Memory == nil ||
   ocispec.Linux.Resources.Memory.Limit == nil {
	return nil
}
```